content/docs/concepts/traffic-management/overview/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Overview and Terminology
+description: Learn about Pilot, Istio's core traffic management component, Envoy proxies, and how they enable service discovery and load balancing.
+weight: 1
+keywords: [traffic-management, pilot, envoy-proxies, service-discovery, load-balancing]
+---
+
+With Istio, you can manage [traffic routing](/docs/concepts/traffic-management/routing)
+and [load balancing](/docs/concepts/traffic-management/overview/#load-balancing)
+for your service mesh without having to update your services. Istio simplifies
+configuration of service-level properties like timeouts and retries, and makes
+it straightforward to set up tasks like staged rollouts with percentage-based
+traffic splits.
+
+Istio's traffic management model relies on the following two components:
+
+-  Pilot, the core traffic management component.
+-  Envoy proxies, which enforce configurations and policies set through Pilot.
+
+These components enable the following high-level tasks:
```

Comment:
again; I think Task is a term that has a meaning in the docs, but isn't commonly how I would think of this in the process of managing a cluster